src/features/tags/components/TagManager/components/TagDialog/index.tsx

```diff
@@ -153,7 +157,9 @@ const TagDialog: React.FunctionComponent<TagDialogProps> = ({
             onCancel={closeAndClear}
             submitDisabled={!title || !color.valid}
             submitText={
-              editingTag ? undefined : messages.dialog.submitCreateTagButton()
+              editingTag || onlyCreateTags
+                ? undefined
+                : messages.dialog.submitCreateTagButton()
```

Comment:
With the changes I've suggested above, this would become `submitLabel` from the properties, and the user of the component could pass in a value if they want anything other than a default "CREATE".

This means that where the tag widget is used on the person page and journey instance page, the label will need to change, and on the new "tags" page, it can be the default.